packages/insured-bridge-relayer/src/Relayer.ts

```diff
@@ -461,167 +460,28 @@ export class Relayer {
-          relayAncillaryDataHash: receipt.events.DepositRelayed.returnValues.relayAncillaryDataHash,
-          proposerBond: receipt.events.DepositRelayed.returnValues.relay.proposerBond,
-          finalFee: receipt.events.DepositRelayed.returnValues.relay.finalFee,
-          depositHash: receipt.events.RelaySpedUp.returnValues.depositHash,
-          instantRelayer: receipt.events.RelaySpedUp.returnValues.instantRelayer,
-          realizedLpFeePct: receipt.events.RelaySpedUp.returnValues.relay.realizedLpFeePct,
-          relayId: receipt.events.RelaySpedUp.returnValues.relay.relayId,
-          relayState: receipt.events.RelaySpedUp.returnValues.relay.relayState,
-          priceRequestTime: receipt.events.RelaySpedUp.returnValues.relay.priceRequestTime,
-          slowRelayer: receipt.events.RelaySpedUp.returnValues.relay.slowRelayer,
-          transactionConfig,
-        });
-      else throw receipt;
-    } catch (error) {
-      this.logger.error({
-        at: "InsuredBridgeRelayer#Relayer",
-        message: "Something errored instantly relaying!",
-        error,
-      });
-    }
-  }
-
-  private async settleRelay(deposit: Deposit, relay: Relay) {
-    await this.gasEstimator.update();
-    try {
-      const { receipt, transactionConfig } = await runTransaction({
-        web3: this.l1Client.l1Web3,
-        transaction: this.generateSettleRelayTx(deposit, relay),
-        transactionConfig: { ...this.gasEstimator.getCurrentFastPrice(), from: this.account },
-        availableAccounts: 1,
-      });
-      if (receipt.events)
-        this.logger.info({
-          at: "InsuredBridgeRelayer#Finalizer",
-          message: "Relay settled 💸",
-          tx: receipt.transactionHash,
-          depositHash: receipt.events.RelaySettled.returnValues.depositHash,
-          caller: receipt.events.RelaySettled.returnValues.caller,
-          realizedLpFeePct: receipt.events.RelaySettled.returnValues.relay.realizedLpFeePct,
-          proposerBond: receipt.events.RelaySettled.returnValues.relay.proposerBond,
-          finalFee: receipt.events.RelaySettled.returnValues.relay.finalFee,
-          relayId: receipt.events.RelaySettled.returnValues.relay.relayId,
-          relayState: receipt.events.RelaySettled.returnValues.relay.relayState,
-          priceRequestTime: receipt.events.RelaySettled.returnValues.relay.priceRequestTime,
-          slowRelayer: receipt.events.RelaySettled.returnValues.relay.slowRelayer,
-          transactionConfig,
-        });
-      else throw receipt;
-    } catch (error) {
-      this.logger.error({ at: "InsuredBridgeRelayer#Finalizer", message: "Something errored settling relay!", error });
-    }
+  private async _generateSettleTransactionForSettleableRelay(deposit: Deposit, relay: Relay) {
+    return {
+      transaction: this._generateSettleRelayTx(deposit, relay),
+      message: "Relay settled 💸",
+      mrkdwn: this._generateMarkdownForSettle(deposit, relay),
```

Comment:
rather, we simply define a `message` and some `mrkdwn`. This information is far more rich and detailed than before, in a more ocnsumable way than it was.